### DISCUSSION TITLE
Import font from google fonts

### DISCUSSION THREAD

**Discussion by 0xdeadc0d3** [vanilla-extract-css/vanilla-extract#600] [Q&A]
How to import font from google fonts? In CSS I can use @import rule. Is there some alternative in vanilla-extract?

**Comment by y-nk** [comment]
I think you need to load your fonts on the side with a `<link>` tag

**Comment by barnabasJ** [comment]
If you check the content of the google font @import URL e.g. https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap you can see that it is just a couple of @font-face rules. You can create those yourselves with the `globalFontFace` function. This has the added benefit of reducing chaining requests to get to your fonts.

**Reply by graup** [reply]
I'm not sure that's a good solution, I guess Google automatically detects which files to import based on the browser.

**Comment by graup** [comment]
@mattcompiles how about adding a low-level primitive like `globalRawStyle` which just appends whatever CSS you want to the output? Or a bit more high-level `globalImport`?

**Reply by graup** [reply]
https://github.com/seek-oss/vanilla-extract/pull/662
